### ISSUE TITLE
Please add availability to use SD checkpoints as main model.

### ISSUE THREAD

**Issue by GodFazer** [lllyasviel/Fooocus#2104]
### Discussed in https://github.com/lllyasviel/Fooocus/discussions/2103

<div type='discussions-op-text'>

<sup>Originally posted by **GodFazer** January 30, 2024</sup>
I had tried to use many of SDXL models but often they don`t give result as some of SD. Fooocus is my fav GUI, so it`d be very nice if we`d have that option to make more realistic and efficient images. Plus it would let us use many interesting loras that not available for SDXL. 

Guys if u agree please show support by ur reactions to show developers importance of improvement</div>

**Comment by mashb1t** [comment]
As Fooocus is a SDXL only GUI with specific adjustments,novelty functions and individual controlnet / adapters his would lead to weeks/months of effort if even possible to guarantee the same level of quality currently offered by the tool.
It is not impossible that Fooocus will support SD 1.5/2.1 somewhen, but for now i am not aware of any plans to do so.

**Comment by kubilaykilinc** [comment]
You can use any SD 1.5 model as refiner. It works perfectly.

**Comment by mashb1t** [comment]
@kubilaykilinc yeah, i briefly forgot about that ^^ This is indeed possible, but 1.5 models can't be used as base model.

**Comment by barepixels** [comment]
If you want to use SD1.5 with Fooocus UI then use this older fork   https://github.com/MoonRide303/Fooocus-MRE  It has not been update in a while
